discussion/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import (Discussion,DiscussionComment,DiscussionVote)
+
+
+admin.site.register(Discussion)
+admin.site.register(DiscussionComment)
+admin.site.register(DiscussionVote)
```

Comment:
Everything good but its always a good practice to add empty line at the end of file

Reply:
Alright that's noted, thanks